test/evergreen.yml

```diff
@@ -525,9 +530,9 @@ functions:
         config=../../../test/format/CONFIG.replay
         for i in $(seq ${times}); do
           echo Iteration $i/${times}
-          x2=$RANDOM$RANDOM
-          x3=$RANDOM$RANDOM
-          rm -rf RUNDIR_{1,2,3}
```

Comment:
Evergreen `shell.exec` uses /bin/sh.  That means that `$RANDOM` is not available, and also the `{a,b,c}` syntax is not expanded.